src/sync/sync_manager.hpp

```diff
@@ -105,18 +105,19 @@ friend class SyncSession;
     // Get a sync user for a given identity, or create one if none exists yet, and set its token.
     // If a logged-out user exists, it will marked as logged back in.
     std::shared_ptr<SyncUser> get_user(const std::string& identity,
-                                       std::string refresh_token,
-                                       util::Optional<std::string> auth_server_url=none,
-                                       SyncUser::TokenType token_type=SyncUser::TokenType::Normal);
-    // Get an existing user for a given identity, if one exists and is logged in.
-    std::shared_ptr<SyncUser> get_existing_logged_in_user(const std::string& identity) const;
+                                       const std::string& auth_server_url,
+                                       std::string refresh_token);
+
+    // Get an admin token user for the given identifier. 
+    std::shared_ptr<SyncUser> get_admin_token_user(const std::string&, std::string);
```

Comment:
The parameters need names since otherwise it's impossible to tell what should be passed.